README.md

```diff
@@ -1,2 +1,4 @@
 # exe_mg
 cow006
+This is a my first code in mipt.
```

Comment:
hey this needs to be changed to for loop not while loop

Reply:
okay, good idea!

Reply:
i'm happy